Bottom: 4389e5ccd819a6d0803f1527aeca1074994ff4ce
Top:    3dc2d85df261635ce775f5736dbd30738699fc9a
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-07-18 13:40:50 -0700

Refresh of rst-afib

---

diff --git a/src/rsocket.c b/src/rsocket.c
index abdd392..1a6d989 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -1063,7 +1063,7 @@ int rsocket(int domain, int type, int protocol)
 	struct rsocket *rs;
 	int index, ret;
 
-	if ((domain != PF_INET && domain != PF_INET6) ||
+	if ((domain != AF_INET && domain != AF_INET6 && domain != AF_IB) ||
 	    ((type != SOCK_STREAM) && (type != SOCK_DGRAM)) ||
 	    (type == SOCK_STREAM && protocol && protocol != IPPROTO_TCP) ||
 	    (type == SOCK_DGRAM && protocol && protocol != IPPROTO_UDP))
